pygfx/objects/_base.py

```diff
@@ -1,7 +1,9 @@
+from __future__ import annotations
```

Comment:
Are there Python versions that we support that need this?

Reply:
No, but in this file it lets you remove the quotes from the self-referencing hints for WorldObject. It’s something I’ve largely gotten in the habit of (unless I know I need runtime type inspection).  If it’s something you’d prefer to use only when absolutely necessary though, I can do that too

Reply:
Oh wait, actually probably yes: I think I used the pipe instead of Union, which is 3.10+

Reply:
> it lets you remove the quotes from the self-referencing hints for WorldObject

Ah, right. Now I remember we actually use it in wgpu-py too. Precisely to avoid quotes.